### ISSUE TITLE
Unable to find resource ID #0x7fa66cf6

### ISSUE THREAD

**Issue by addy419** [fast4x/RiMusic#3834]
### Steps to reproduce the bug

Open RiMusic and play a song

### Expected behavior

Not crash.

### Actual behavior

Crashes after 1-2 seconds.

### Logs

[RiMusic log 48f02b866ad0.txt](https://github.com/user-attachments/files/17197506/RiMusic.log.48f02b866ad0.txt)

### RiMusic version

0.6.54.1

### Android version

Android 14 - GrapheneOS

### Additional information

Started happening after the last update 2 days ago.

**Comment by darkknight7777777** [comment]
I'm seeing the same error, also running grapheneos v14

**Comment by Jacoblightning** [comment]
Same happened for me. I tolled back and updated again. Sadly, crashes have returned after a reboot.

**Comment by Jacoblightning** [comment]
I wonder if this is YouTube fighting back or RiMusic

**Comment by knighthat** [comment]
not really YouTube's fault. If it's YT, the player will crash but not the whole app. Please go into your Settings > Misc > (scroll down) Copy log to clipboard > Paste it here

**Comment by fast4x** [comment]
Bug not reproducible

**Comment by Jacoblightning** [comment]
I have a logcat here: `FATAL EXCEPTION: main
Process: it.fast4x.rimusic, PID: 19488
android.content.res.Resources$NotFoundException: Unable to find resource ID #0x7fa66cf6
	at android.content.res.ResourcesImpl.getResourcePackageName(ResourcesImpl.java:314)
	at android.content.res.Resources.getResourcePackageName(Resources.java:2353)
	at it.fast4x.rimusic.service.PlayerMediaBrowserService.e(SourceFile:19)
	at V7.o.r(SourceFile:343)
	at B8.a.i(SourceFile:9)
	at V8.O.run(SourceFile:113)
	at K5.q.run(SourceFile:232)
	at d9.j.run(SourceFile:3)
	at d9.a.run(SourceFile:96)`.

I downgraded so I can't get the full log right now.

**Comment by Jacoblightning** [comment]
Weird, I just upgraded again and it's working. Will post full log if it crashes again.

**Comment by knighthat** [comment]
it is possible that a file got corrupted. Update (reinstall) the app replaces the corrupted file so it works fine I guess

**Comment by addy419** [comment]
@knighthat Here's the log.

```
2024-10-02 06:34:49:798 DEBUG: MainApplication - Log enabled at /data/user/0/it.fast4x.rimusic/files/logs
2024-10-02 06:34:52:769 INFO: C - PlayerService createDataSourceResolverFactory dataSpec DataSpec[GET GFUiZJN906s, 0, -1, GFUiZJN906s, 6]
2024-10-02 06:34:52:770 INFO: C - PlayerService createDataSourceResolverFactory dataSpec isLocal false videoId GFUiZJN906s all DataSpec[GET GFUiZJN906s, 0, -1, GFUiZJN906s, 6]
2024-10-02 06:34:52:770 INFO: C - PlayerService createDataSourceResolverFactory dataSpec isInCache true
2024-10-02 06:34:52:770 INFO: C - PlayerService createDataSourceResolverFactory dataSpec isInDownloadCache false
2024-10-02 06:34:52:770 INFO: C - PlayerService createDataSourceResolverFactory dataSpec videoId == ringBuffer0 false
2024-10-02 06:34:52:771 INFO: C - PlayerService createDataSourceResolverFactory dataSpec videoId == ringBuffer1 false
2024-10-02 06:34:57:059 DEBUG: MainApplication - Log enabled at /data/user/0/it.fast4x.rimusic/files/logs
2024-10-02 06:34:58:882 DEBUG: MainApplication - Log enabled at /data/user/0/it.fast4x.rimusic/files/logs
2024-10-02 06:35:04:892 DEBUG: MainApplication - Log enabled at /data/user/0/it.fast4x.rimusic/files/logs
2024-10-02 06:35:07:272 DEBUG: s - CheckMonthlyPlaylist false
2024-10-02 06:35:07:335 DEBUG: s - CheckMonthlyPlaylist true
2024-10-02 06:35:07:335 DEBUG: s - SongsMostPlayed null
2024-10-02 06:35:07:415 DEBUG: s - CheckMonthlyPlaylist true
2024-10-02 06:35:07:415 DEBUG: s - SongsMostPlayed 98
```

And this is the crash log.
```
2024-10-02T06:34:53.342:	android.content.res.Resources$NotFoundException: Unable to find resource ID #0x7fa66cf6
	 android.content.res.ResourcesImpl.getResourcePackageName(ResourcesImpl.java:314) 
	 android.content.res.Resources.getResourcePackageName(Resources.java:2363) 
	 it.fast4x.rimusic.service.PlayerMediaBrowserService.e(SourceFile:19) 
	 V7.o.r(SourceFile:343) 
	 B8.a.i(SourceFile:9) 
	 V8.O.run(SourceFile:113) 
	 K5.q.run(SourceFile:232) 
	 d9.j.run(SourceFile:3) 
	 d9.a.run(SourceFile:96) 

2024-10-02T06:34:57.373:	android.content.res.Resources$NotFoundException: Unable to find resource ID #0x7fa66cf6
	 android.content.res.ResourcesImpl.getResourcePackageName(ResourcesImpl.java:314) 
	 android.content.res.Resources.getResourcePackageName(Resources.java:2363) 
	 it.fast4x.rimusic.service.PlayerMediaBrowserService.e(SourceFile:19) 
	 V7.o.r(SourceFile:343) 
	 B8.a.i(SourceFile:9) 
	 V8.O.run(SourceFile:113) 
	 K5.q.run(SourceFile:232) 
	 d9.j.run(SourceFile:3) 
	 d9.a.run(SourceFile:96) 

2024-10-02T06:35:00.764:	android.content.res.Resources$NotFoundException: Unable to find resource ID #0x7fa66cf6
	 android.content.res.ResourcesImpl.getResourcePackageName(ResourcesImpl.java:314) 
	 android.content.res.Resources.getResourcePackageName(Resources.java:2363) 
	 it.fast4x.rimusic.service.PlayerMediaBrowserService.e(SourceFile:19) 
	 V7.o.r(SourceFile:343) 
	 B8.a.i(SourceFile:9) 
	 V8.O.run(SourceFile:113) 
	 K5.q.run(SourceFile:232) 
	 d9.j.run(SourceFile:3) 
	 d9.a.run(SourceFile:96) 
```

**Comment by fast4x** [comment]
Yes just founded cause, fixed and available from version 0.6.55

**Comment by fast4x** [comment]
try custom version https://github.com/fast4x/RiMusic/tree/master/customVersion
